apps/web/src/routes/orders/order-list/components/forms/order-form.tsx

```diff
@@ -289,7 +294,9 @@ const OrderItems = ({ form }: { form: UseFormReturn<OrderFormData> }) => {
 
           {fields.map((field, index) => (
             <TableRow key={field.id}>
-              <TableCell>{selectedVariants[index]?.name}</TableCell>
+              <TableCell>
+                {selectedVariants[index]?.product?.name || field.variant?.product?.name || 'Unknown Produt'}
```

Comment:
I think both product name and variant are important. 

Plus why is index based product and the field based product name? Can we clean this?